core/src/main/java/io/questdb/cairo/RecordChain.java

```diff
@@ -111,6 +112,11 @@ public long getAddress(long recordOffset, int columnIndex) {
         return addressOf(getOffsetOfColumn(recordOffset, columnIndex));
     }
 
+    public Record cloneRecord(long recordOffset) {
```

Comment:
`cloneRecord` name is a bit misleading. I'd better call it something like `getRecordAt`